authors/apps/profiles/models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+from django.db.models.signals import post_save, pre_save
+from django.utils.text import slugify
+from cloudinary.models import CloudinaryField
+from ..authentication.models import User, UserManager
+
+
+class UserProfile(models.Model):
+    """
+    This class contains  models relating to user profile. 
+    It has a one to one relationship with User model class.
+    An intance of this class is created during user registration by calling 
+    the create profile in the post_save function
+    """
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    # A char field to hold bio data
+    bio = models.TextField(max_length=200, default='empty')
+
+    # A choice field to hold multiple choice
+    gender_choices = (('M', 'MALE'), ('F', 'FEMALE'),
+                      ('N', 'I PREFER NOT TO SAY'))
+    gender = models.CharField(
```

Comment:
Why not leave as blank until I select a gender

Reply:
I thought it would be user-friendly to ask for clarification for the user. So the user knows they don't have to provide a gender